src/stylesheets/mixins/next-icon-container.scss

```diff
@@ -0,0 +1,17 @@
+@mixin next-icon-container() {
+  padding: rem(14px 13px);
+  position: relative;
```

Comment:
<!--
  0 failure: 
  2 warning:  
	  Expected "pos..., Expected an empty...
  
  
  DangerID: danger-id-Danger;
  File: src/stylesheets/mixins/next-icon-container.scss;
  Line: 3;
-->

- :warning: 
	  Expected "position" to come before "padding" in group "Positioning" (order/properties-order)
- :warning: Expected an empty line before property "position" (order/properties-order)